pkg/machines/libvirt.es6

```diff
@@ -332,6 +324,53 @@ LIBVIRT_PROVIDER = {
         };
     },
 
+    SET_VCPU_SETTINGS ({ name, connectionName, count, max, sockets, cores, threads, isRunning }) {
+        logDebug(`${this.name}.SET_VCPU_SETTINGS(${name}):`);
+        return dispatch => spawnVirshReadOnly({
+            connectionName,
+            method: 'dumpxml',
+            name
+        }).then((domXml) => {
+
+            const domainElem = getDomainElem(domXml);
+
+            let cpuElem = domainElem.getElementsByTagName("cpu")[0];
```

Comment:
No need to do it now, but following code shall be added to a function to keep `SET_VCPU_SETTINGS` smaller

Reply:
Rebase is needed anyway, so can you please share the domain xml parsing with `parseDumpxml`? Will help to keep the code smaller and to the point.